js/ui/grid_core/ui.grid_core.columns_resizing_reordering.js

```diff
@@ -575,12 +575,17 @@ const ColumnsResizerViewController = modules.ViewController.inherit({
         return false;
     },
 
+    _isRtlMode: function() {
```

Comment:
_isRtlMode -> _isRtlParentStyle

Reply:
Also, I think that we should check this style only if rtlEnabled is true to improve performance without rtl